### PR TITLE
Refactor Contact and Portfolio

### PR DESCRIPTION
**Pull request by king-11**
## Description

- fix: sizing use rem not irregular px
- refactor & style portfolio page
- sematic HTML used in portfolio page
- add alum data
- make year dynamic instead of hard coded

- #45 

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by Manan-Rathi — DISMISSED** [review]
LGTM!

File: components/about-us/aboutuslogo.vue [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="header">
-    <div class="flex flex-row justify-center items-center w-full main py-100px">
+    <div class="flex flex-row justify-center items-center w-full main py-24">
       <h1 class="aboutus">About Us</h1>
     </div>
     <div>
       <svg
-        class="waves h-15/100 w-full min-h-100px max-h-150px"
+        class="waves h-15/100 w-full min-h-100 max-h-36"
```
Comment (debarati-06) [inline components/about-us/aboutuslogo.vue:8]: Maybe we can reduce the height in About Us title animation part as the waves are not visible in single screen space and scrolling is required.

**Review by debarati-06 — COMMENTED** [review]
Rest everything looks great!

File: components/about-us/aboutuslogo.vue (hunk 1 above)
Comment (king-11) [reply components/about-us/aboutuslogo.vue:8]: Yep that can be good on it

**Review by somesh202 — DISMISSED** [review]
LGTM.

File: components/about-us/aboutuslogo.vue (hunk 1 above)
Comment (king-11) [reply components/about-us/aboutuslogo.vue]: @debarati-06 can you take a look now

Comment (debarati-06) [reply components/about-us/aboutuslogo.vue]: Yes, it looks great!✨

**Pull request merged**